boards/arm64/fvp_base_revc_2xaemv8a/doc/index.rst

```diff
@@ -71,6 +71,30 @@ To run with FVP, ARMFVP_BIN_PATH must be set before running:
 
 e.g. export ARMFVP_BIN_PATH=<path/to/fvp/dir>
 
+Running Zephyr at EL1NS
+***********************
+
+It may be needed to run Zephyr at EL1NS level with ``CONFIG_ARMV8_A_NS``
+In order to run at EL1NS, you'll need a proper Firmware loaded in the FVP model
+to run Zephyr at non-secure EL3.
```

Comment:
This statement is confusing. You want to run Zephyr at EL1NS, while TF-A is running at EL3. There is no such a thing as non-secure EL3.

Reply:
Yes, Carlo is right.
EL3 is secure only.  https://developer.arm.com/documentation/102412/0100/Execution-and-Security-states

Reply:
Hi, sorry it’s a typo, BL33 obviously runs at EL1

Reply:
I reworded the text, it meant nothing as-is...